modules/civiremote_event/src/Element/Confirm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/*
+ * Copyright (C) 2023 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types=1);
+
+namespace Drupal\civiremote_event\Element;
+
+use Drupal\Core\Render\Element\FormElement;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a form element for double-input of values.
+ *
+ * Formats as a pair of fields, which do not validate unless the two entered
+ * values match.
+ *
+ * Properties:
+ * - #field: The render array of the form field to confirm.
```

Comment:
This should be restricted to allow form elements only, as otherwise this could take any render array, which makes validating difficult.

Reply:
I've renamed `#field` to `#element` so it should be more obvious that it is meant to be used for form elements.

Reply:
See https://github.com/systopia/civiremote/pull/2#discussion_r1200272414